### DISCUSSION TITLE
Add passhole password manager as an alternative for KeePass users

### DISCUSSION THREAD

**Discussion by JackTheMico** [twpayne/chezmoi#2098] [Ideas]
Can we add [passhole](https://github.com/Evidlo/passhole) as an alternative integration for KeePass users?
I'm willing to implement this feature.

**Comment by twpayne** [comment]
Yes, this would be great. A PR to implement this would be very welcome. https://github.com/twpayne/chezmoi/pull/2062/files will give you a good idea of how to implement this.

I'll create an issue from this discussion and assign it to you.
